scripts/installer/__init__.py

```diff
@@ -0,0 +1,12 @@
+import sys
+
+sys.dont_write_bytecode = True
+
+# https://stackoverflow.com/questions/3365740/how-to-import-all-submodules
+import pkgutil
+
+__all__ = []
+for loader, module_name, is_pkg in pkgutil.walk_packages(__path__):
+    __all__.append(module_name)
+    _module = loader.find_module(module_name).load_module(module_name)
```

Comment:
I swapped this line for `_module = loader.find_spec(module_name)`  based on internet help because I am using Python 3.12 and I got errors that `find_module` and `load_module` are deprecated. I am not 100% if thats the best/correct solution but it worked.

Reply:
Tested patch and confirmed. I can invoke packages directly from CLI and import `scripts.installer` from an interactive session.